src/Tribe/Log/File_Logger.php

```diff
@@ -64,7 +64,7 @@ protected function obtain_handle() {
 			touch( $this->log_file );
 		}
 
-		if ( is_readable( $this->log_file ) ) {
+		if ( is_readable( $this->log_file ) && is_writable( $this->log_file ) ) {
```

Comment:
What about something like:

```
$is_writable_if_needed = 'r' !== $this->context && is_writable( $this->log_file );

if ( is_readable( $this->log_file ) && $is_writable_if_needed ) {
    /* ... existing code */
}
```

That variable name may need some work, but my thinking is that if the log file is opened in _read_ mode, rather than _append,_ we don't actually care about writability and in fact we'd want to open it even if it is not writable.